scripts/docker_publish_dev.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+tag=$([ ! "${TRAVIS_PULL_REQUEST}" = "false" ] && echo "${TRAVIS_PULL_REQUEST}" || echo "${TRAVIS_BRANCH}")
```

Comment:
You don't want tag to be created if there's no PR?

Reply:
I do want it created but with the branch name

Reply:
So don't we need to add parentheses? just to make it clear?
```tag=$(([ ! "${TRAVIS_PULL_REQUEST}" = "false" ] && echo "${TRAVIS_PULL_REQUEST}") || echo "${TRAVIS_BRANCH}")```

Reply:
Oh, OK I can try... but it's bash :) not sure if it will work as expected

Reply:
So don't. Maybe it was just hard for me to understand